msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Common.targets

```diff
@@ -263,7 +263,7 @@ Copyright (C) 2013-2016 Xamarin. All rights reserved.
 	</Target>
 
 	<!-- Insert our app bundle generation step -->
-	<PropertyGroup>
+	<PropertyGroup Condition="'$(_UsingXamarinSdk)' != 'true'">
```

Comment:
@rolfbjarne why are we skipping this property group for .NET 5?

Reply:
Eventually we'll do the same thing (inject our targets into BuildDependsOn), but there will be significant differences, and it's easier to add targets one-by-one while the differences are implemented, instead of fixing everything at the same time.